src/templates/base/2019/accessibility_statement.html

```diff
@@ -35,12 +35,14 @@ <h2 class="header">
     </div>
   </nav>
 
-  <article id="maincontent" class="content">
-    <h1 class="title title-lg">Accessibility Statement</h1>
-      
-    {{ self.main_content() }}
-
-  </article>
+  <div class="content">
```

Comment:
This looks redundant, having two .content elements. Why do we need the extra div?

Reply:
Will remove the class from the article.

The extra div is needed to make it similar to chapters so it gains the 10px margin-left mentioned about:

```css
.content > section, .content > article {
  margin-bottom: 32px;
  margin-left: 10px;
}
```

Chapters need the two divs because they have the article, and then the extra info (discussion/query/results links, author links, prev/next). Technically the accessibility page doesn't need this but would prefer to add for consistency, rather than treat specially.

The Methodology page doesn't use this extra div, but works because it's divided into sections which aren't really needed (the chapters are not divided into `<sections>`), and are a bit of an anomaly compared to chapters. Now you've pointed this out I think I'd like to clean this up as part of this and add the extra div to methodology as well and remove the `<sections>` - again to be consistent with chapters. Plus `<section>` is fairly meaningless to be honest: https://www.smashingmagazine.com/2020/01/html5-article-section/

> In most screen readers, a `<section>` element isn’t announced unless it has an accessible name. In this case, the text of the aria label.
>
>...
>
>`<section`> can be used with aria-label to signal to a screen reader user where a particular sub-part of an article begins and ends. Otherwise, forget about it, or use another element, such as `<aside aria-label="quick summary">` or `<div role=”region” aria-label="quick summary">`.

Reply:
Changed my mind and added `<section>` elements to accessibility statement to bring it into line with Methodology. So have removed the extra div.